src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -302,6 +301,17 @@ let add_breadcrumb_exn t breadcrumb =
         , `Int (List.length @@ Full_frontier.all_breadcrumbs t.full_frontier)
         ) ]
     "POST: ($state_hash, $n)" ;
+  let user_cmds =
+    Breadcrumb.user_commands @@ Full_frontier.best_tip t.full_frontier
```

Comment:
I think this is a bug. I'm pretty sure you meant to do `Breadcrumb.user_commands breadcrumb` so that you print out the user commands of the breadcrumb you just added.